### DISCUSSION TITLE
erase and redo the indexing of faces

### DISCUSSION THREAD

**Discussion by MrLafia** [photoprism/photoprism#1565] [Q&A] [answered]
I made a big mistake by merging two faces
is there a way to remove all face tags
and redo a complete scan

**Comment by lastzero** [comment]
Yes, see Getting Started docs.

**Comment by MrLafia** [comment]
wow it's been a long time since I had the famous answer "RTFM"
ok, i'm going to go read the whole manual again

**Reply by lastzero** [reply]
I'm AFK and still wanted to give you a quick answer. It's 9pm over here. Do you have a problem with this?

**Comment by jackbrown1993** [comment, marked as answer]
Facial Recognition

Existing users may index faces in originals without performing a complete rescan:

docker-compose exec photoprism photoprism faces index

For a fresh start e.g. after upgrading from a development preview, remove known people and faces before re-indexing:

docker-compose exec photoprism photoprism faces reset -f

**Reply by lastzero** [reply]
Thanks!

**Comment by MrLafia** [comment]
Thank you very much, I was able to erase all the faces
the rescan is in progress

**Reply by lastzero** [reply]
You're welcome! 👍
